pkg/compliance/cgroups_nolinux.go

```diff
@@ -7,10 +7,10 @@
 
 package compliance
 
-func getProcessContainerID(pid int32) (string, bool) {
+func getProcessContainerID(int32) (string, bool) {
```

Comment:
I'm sorry but this feels like we are losing information here (and in many other similar places), I don't think we should keep this linter if it makes things worse than they are

Reply:
I might be able to change to using `_paramName` according to `unused params` docs, let me try :cat_typing:

Reply:
Current latest `golangci-lint` does uses `revive` `v1.3.2`, and does not support `unused-parameter` configuration that comes with `v1.3.4`, so we cannot allow a `^_` pattern, like
```golang
func getProcessContainerID(_pid int32) (string, bool) {
```

I suggest we disable this specific linter in the meantime, adding a comment to enable it once `revive` is bumped to `v1.3.4`

Reply:
The normal way to mark an unused parameter is a single underscore `_` for the name.

Reply:
I took another road, that we'll test this time and, if it works for everyone, we'll use on any new linter rule: we'll take care of fixing only quick wins, and mute the linter per line in the bump/update PR. Each muted line will have the same comment 
```golang
// nolint:[linter]// TODO fix [linter] [rule]
```
We will create a card per code owner and adapt tools to help removing `//nolint` on owned files, based on the bump PR, so that teams can work on their fixes.